app/src/main/java/org/inspire/breath/data/AppRoomDatabase.java

```diff
@@ -5,9 +5,12 @@
 import android.arch.persistence.room.RoomDatabase;
 import android.content.Context;
 
-@Database(entities = {Patient.class}, version = 1)
+@Database(entities = {Patient.class, Recording.class}, version = 2)
 public abstract class AppRoomDatabase extends RoomDatabase {
-    public abstract MainDao mainDao();
+
+    // DAOs
+    public abstract PatientDao patientDao();
```

Comment:
naming inconsistency between the DAOs (patientDao, recordingDAO), should change to recordingDao to match class name I think

Reply:
Yep, well spotted!